help/help.adoc

```diff
@@ -8,29 +8,56 @@ Revised on May 18, 2020
 
 This plugin was developed and tested against Windows Azure 2012.
 
-{CD} minimum version 4.0
+{PRODUCT} minimum version 4.0
 
 [[Windows_Certificates]]
-== Windows certificates
+== Working with Windows certificates
 
-Some procedures require a certificate to authenticate the request, you need to create one and upload it to Windows Azure:
+Azure procedures may require a Windows certificate to authenticate requests. To create and upload a certificate to Windows Azure:
 
-[[Create_a_X509_certificate]]
-== Create a x509 certificate
+. Generate an X.509 certificate using the Certificate Creation Tool (Makecert.exe) using the follow command:
++
+`makecert -sky exchange -r -n "CN=<CertificateName>" -pe -a sha1 -len 2048 -ss My "D:/<CertificateName>.cer"`
++
+NOTE: Makecert.exe is included with Visual Studio and Windows SDK installations. For more information, refer to link:http://msdn.microsoft.com/en-us/library/bfsktky3(VS.80).aspx[Microsoft's Makecert.exe page].
 
-Using Certificate Creation Tool (Makecert.exe): makecert -sky exchange -r -n "CN=<CertificateName>" -pe -a sha1 -len 2048 -ss My "D:/<CertificateName>.cer"
+. To upload the Windows certificate to Windows Azure, refer to link:https://azure.microsoft.com/en-us/documentation/articles/cloud-services-configure-ssl-certificate/[How to Add a Management Certificate to a Windows Azure Subscription].
```

Comment:
Sorry; should have mentioned this yesterday. If this is the correct link, recommend updating the link title to match the title on the page that we're linking to.
```suggestion
. To upload the Windows certificate to Windows Azure, refer to link:https://azure.microsoft.com/en-us/documentation/articles/cloud-services-configure-ssl-certificate/[Configuring TLS for an application in Azure].
```

Reply:
updated